packages/builder/src/builtin-tasks/run.ts

```diff
@@ -17,23 +23,40 @@ function filterNonExistent (scripts: string[]): string[] {
   return scripts.filter(script => !fsExtra.pathExistsSync(script));
 }
 
-export async function runMultipleScripts (runtimeEnv: AlgobRuntimeEnv,
+export async function runMultipleScripts (
+  runtimeEnv: AlgobRuntimeEnv,
   scriptNames: string[],
-  runScriptFn: (
-    relativeScriptPath: string,
-    runtimeEnv: AlgobRuntimeEnv
-  ) => Promise<void>): Promise<void> {
+  onSuccessFn: (cpData: CheckpointData, relativeScriptPath: string) => void,
+  force: boolean,
+  logTag: string,
+  wrapDeployer: (orig: AlgobDeployer) => AlgobDeployer): Promise<void> {
+  const log = debug(logTag);
+  const cpData: CheckpointData = loadCheckpointsRecursive();
```

Comment:
no need to add a type here. The function has a clear return type

Reply:
It is clear but I prefer to have it in this place.